contracts/StablecoinConverter.sol

```diff
@@ -224,15 +227,15 @@ contract StablecoinConverter is EpochTokenLocker {
         uint batchIndex,
         address[] memory owners,  //tradeData is submitted as arrays
         uint16[] memory orderIds,
-        uint128[] memory volumes,
+        uint128[] memory remainingAmounts,
         uint16[] memory tokenIdsForPrice
     ) internal {
         previousSolution.batchId = batchIndex;
         for (uint i = 0; i < owners.length; i++) {
             previousSolution.trades.push(TradeData({
                 owner: owners[i],
                 orderId: orderIds[i],
-                volume: volumes[i]
+                remainingAmount: remainingAmounts[i]
```

Comment:
Here as well, if I understand correctly we are storing the volume and not the remaining amount.